Podfile.lock

```diff
@@ -86,6 +95,6 @@ SPEC CHECKSUMS:
   Tabman: 627b2bddc6c054bcd86c60919000c30c4adf2ec4
   TUSafariActivity: afc55a00965377939107ce4fdc7f951f62454546
 
-PODFILE CHECKSUM: ace2f34a7d73e2c1c9cdfc7471f86b2916e4cc49
+PODFILE CHECKSUM: c1b736f34980e21e7389fb17ca6896cf06d018ed
 
-COCOAPODS: 1.2.1
+COCOAPODS: 1.3.1
```

Comment:
Not part of this PR, but consider using NPM/bundler to ensure everyone is using same version of CP to prevent issues

Reply:
Ooh, whoops. I will see if I can change back to `1.2.1`

Reply:
Don't worry about it, I'm pretty sure Ryan is on 1.3.1 also (It's me at fault!)